clang/test/SemaHLSL/resource_binding_attr_error_space.hlsl

```diff
@@ -0,0 +1,66 @@
+// RUN: %clang_cc1 -triple dxil-pc-shadermodel6.3-library -x hlsl -o - -fsyntax-only %s -verify

```

Comment:
This file seems to now mostly be testing non-error cases of specifying a space for registers.  I think it'd probably be better to just merge these into the other tests we have.

Specifying a register space isn't some bit special thing, it's part of the normal flow we should consider when validating that, say, `u` type registers work with particular types of resource.

Reply:
I will at least say it's simpler for a distinct file to test this, since we test multiple resource types (UDTs, basic, resources) within different scopes.